node/network/bridge/src/action.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) enum Action {
 	SendCollationMessages(Vec<(Vec<PeerId>, protocol_v1::CollationProtocol)>),
 
 	/// Ask network to send requests.
-	SendRequests(Vec<Requests>),
+	SendRequests(Vec<Requests>, IfDisconnected),
```

Comment:
could you elaborate why we need this? always seems to be `IfDisconnected::TryConnect`

Reply:
Not for collator-protocol - there we actually only request from peers we should be connected to via peer set already.